connection/impl.go

```diff
@@ -285,6 +285,8 @@ func createGrpcConnection(conf *Configuration, address string) (*grpc.ClientConn
 				})))
 	}
 
+	log.Printf("[warn] >>> [%v]:[%v]", conf.Username, conf.Password)
```

Comment:
I'm not sure if logging user and password is the best idea :sweat_smile:

Reply:
ahaha some debugging leftovers. Nice catch!